### ISSUE TITLE
Import from parquet does not work

### ISSUE THREAD

**Issue by q0-0n** [datacontract/datacontract-cli#525]
Running the command:

`datacontract import --format parquet --source localpathtoparquetfile.snappy.parquet`

Results in the following error message:

```
Usage: datacontract import [OPTIONS]
Try 'datacontract import --help' for help.
╭─ Error ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ Invalid value for '--format': 'parquet' is not one of 'sql', 'avro', 'dbt', 'dbml', 'glue', 'jsonschema', 'bigquery', 'odcs', 'unity', 'spark', 'iceberg'.                │
╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
```

Expected behavior: creation of local datacontract YAML file.

The value "parquet" for format should be correct. See documentation: https://github.com/datacontract/datacontract-cli?tab=readme-ov-file#import

Version: 0.10.14

**Comment by jochenchrist** [comment]
Support for parquet import will be available with v0.10.15.

We'll do a release shortly.

**Comment by simonharrer** [comment]
Release is now available: https://github.com/datacontract/datacontract-cli/releases/tag/v0.10.15
